models/DataObject/ClassDefinition/Data/QuantityValue.php

```diff
@@ -169,6 +169,26 @@ class QuantityValue extends Data implements ResourcePersistenceAwareInterface, Q
         'unit' => 'varchar(64)',
     ];
 
+    public function __wakeup(): void
+    {
+        $this->init();
+    }
+
+    /**
+     * @internal
+     *
+     * @return $this
+     */
+    protected function init(): static
```

Comment:
I am not sure why all the `init` functions needs to be protected instead of private?